pkg/unvanquished_src.dpkdir/ui/hud_basics.rml

```diff
@@ -181,9 +181,13 @@
 </head>
 <body id="hud_basics">
 	<region class="topright">
-		<if cvar="cg_minimapActive" condition="==" value="1">
-			<minimap />
-		</if>
+		<minimapbox>
```

Comment:
Use `div` with a class name instead of making up a random element name?

Reply:
Done.